### ISSUE TITLE
Sequencing of actions

### ISSUE THREAD

**Issue by owickstrom** [garyb/purescript-indexed-monad#4]
Right now one has to use any of the following to sequence indexed monadic actions and discarding return values:

```purescript
foo :>>= \_ -> bar` -- or `const bar`

foo `mbind` \_ -> bar -- or `const bar`

do foo
   bar
   where bind = ibind
```

If we had `IxApply`, we could implement `:*>`, corresponding to `*>` from Apply. That would take us down the path of implementing the whole type class hierarchy (Bind, Applicative, Apply, Functor, more?).

```purescript
foo :*> bar
```

We could perhaps also add `:>>`, like `>>` in Haskell's prelude, and just have it defined for only for `IxMonad`.

```purescript
foo :>> bar
```

**Comment by owickstrom** [comment]
Could maybe be closed by https://github.com/garyb/purescript-indexed-monad/pull/5?

**Comment by garyb** [comment]
Closed by #5 for now, but I guess `IxApply` etc. will probably be good to have in the long run :)
